README.md

```diff
@@ -72,7 +72,7 @@ see [servers/cromwell](servers/cromwell/README.md#Development).
 #### Notes
 1. Websocket reload on code change does not work in docker-compose (see
 https://github.com/angular/angular-cli/issues/6349).
-1. Changes to `package.json` or `requirements.txt` require a rebuild with:
+1. Changes to `package.json` or `requirements.txt` or [regenerating the API](#updating-the-api-using-swagger-codegen) require a rebuild with:
```

Comment:
nit: `1.` -> `2.` although they are the same thing in rendered markdown file.

Reply:
thanks for pointing that out -- fixed

Reply:
FYI: This was intentional, it's to avoid having to shift every number in the list if you add a new bullet.

Reply:
I was reacting to a comment by @rexwangcc -- I have no strong opinion either way.

Reply:
OK, so I didn't know it was an intentional cookie :). I personally prefer to outline the ordered lists use ordered numbers here. Although all of them will eventually be rendered to "<li></li>" on web pages, having ordered numbers helps me manage the content in vim/sublime. 

I won't be unhappy if we switch back to use all `1.`s though(we need to make sure the first item uses `1.` based on Markdown's spec)